spec/core/client_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+RSpec.describe "Client", :type => :client do
+
+  it "confirm default value initialize" do
+    client = Caplinked::Client.new
+
+    expect(client.api_host).to eq("sandbox.caplinked.com")
+    expect(client.api_scheme).to eq("https")
```

Comment:
default is https, but any value can be passed in. I think its fine, the users that will have access are paying customers anyways.